recipes/django-ses/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "django-ses" %}
+{% set version = "0.8.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ad45fca653db1725d2237d5aa1a4e2705789088717d69b9a91e4a1c7688896b5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - django
+    - boto >=2.31.0
+    - pytz >=2016.10
+    - future >=0.16.0
+
+test:
+  commands:
+    - python -c "import importlib; assert importlib.util.find_spec('django_ses')"  # [py37]
```

Comment:
I don't think you should have a selector for a noarch package.

Does this test not work for python 3.6? let alone 2

Reply:
This is because `importlib.util.find_spec` API is different across python 3 version. So it was an easy quick fix...

Reply:
Is it true that the only difference is the error that is raised?

https://docs.python.org/3/library/importlib.html#importlib.util.find_spec

considering only python 3.6 +

Reply:
I remember there was a bug first time I did it. Let see again.